source/scripting/Vehicle.cs

```diff
@@ -1429,6 +1429,12 @@ public int PassengerCount
 				return Function.Call<int>(Hash.GET_VEHICLE_NUMBER_OF_PASSENGERS, Handle);
 			}
 		}
+        public int GetVehicleWeapon(Ped Ped)
```

Comment:
I think a new enum should be made for VehicleWeaponHash (they are defined in the same files as Weapons) We dont want users mixing WeaponHash and VehicleWeaponHash when say giving a ped a weapon. IIRC there is also a way to set the VehicleWeapon